test/specs.js

```diff
@@ -78,25 +78,47 @@ describe('module-alias', function () {
     expect(something).to.equal('Hello from foo')
   })
 
-  it('should import settings from package.json', function () {
-    moduleAlias({
-      base: path.join(__dirname, 'src')
+  describe('importing settings from package.json', function() {
+    var baseWorkingDirectory
+    beforeEach(function() {
+      baseWorkingDirectory = process.cwd()
     })
 
-    var src, foo, baz, some, someModule
-    try {
-      src = require('@src/foo')
-      foo = require('@foo')
-      baz = require('@bar/baz')
-      some = require('some/foo')
-      someModule = require('some-module')
-    } catch (e) {}
+    afterEach(function(){
+      process.chdir(baseWorkingDirectory)
+    })
 
-    expect(src).to.equal('Hello from foo')
-    expect(foo).to.equal('Hello from foo')
-    expect(baz).to.equal('Hello from baz')
-    expect(some).to.equal('Hello from foo')
-    expect(someModule).to.equal('Hello from some-module')
+    function expectAliasesToBeImported() {
+      var src, foo, baz, some, someModule
+      try {
+        src = require('@src/foo')
+        foo = require('@foo')
+        baz = require('@bar/baz')
+        some = require('some/foo')
+        someModule = require('some-module')
+      } catch (e) {}
+
+      expect(src).to.equal('Hello from foo')
+      expect(foo).to.equal('Hello from foo')
+      expect(baz).to.equal('Hello from baz')
+      expect(some).to.equal('Hello from foo')
+      expect(someModule).to.equal('Hello from some-module')
+    }
+
+    it('should import settings from user-defined base path', function () {
+      moduleAlias({
+        base: path.join(__dirname, 'src')
+      })
+
+      expectAliasesToBeImported()
+    })
+
+    it('should import default settings from process.cwd()', function() {
+      process.chdir(path.join(__dirname, 'src'))
+      moduleAlias()
```

Comment:
How do we know if `process.cwd()` was actually called and it's value was used for the path. Right now I think it uses this path `nodePath.join(__dirname, '../..')` but not the `process.cwd`. Maybe `sinon.stub` it? Like 

```js
var processCwdStub
beforeEach(function () {
  processCwdStub = sinon.stub(process, 'cwd').callsFake(function () { return '/not-existing-path' })
})
afterEach(function () {
  processCwdStub.restore()
})

it('should use path from process.cwd()', function () {
  expect(function () { moduleAlias() }).to.throw('Cannot resolve module /not-existing-path') // or whatever is the copy that comes from `require`
  expect(processCwdStub.callCount).to.eq(1)
})
```

Didn't test it, but you got the point

Reply:
When you remove `process.cwd()` from index.js, this test fails, so I think this test covers the use case.

We can test failures modes of `process.cwd()` returns a "wrong" path, but I wouldn't naturally do it. And we would be pulling Sinonjs in, which we might not want to do. What do you think ?